products/xcp-ng.md

```diff
@@ -0,0 +1,66 @@
+---
+title: XCP-ng
+category: os
+permalink: /xcp-ng
+alternate_urls:
+-   /xcpng
+versionCommand: software-version
+releasePolicyLink: https://xcp-ng.org/docs/releases.html
+
+auto:
+-   git: https://github.com/xcp-ng/xcp-ng-release.git
+
+releases:
+-   releaseCycle: "8.2"
+    releaseDate: 2020-11-18
+    eol: false
```

Comment:
Support column is not needed : a cycle is either fully supported or not supported at all (https://web.archive.org/web/20210508124935/https://xcp-ng.org/docs/releases.html).

Reply:
Column removed